winpr/libwinpr/comm/test/TestCommDevice.c

```diff
@@ -64,7 +64,7 @@ static int test_CommDevice(LPCTSTR lpDeviceName, BOOL expectedResult)
 			return FALSE;
 		}
 
-		if (lpTargetPath[_tcslen(lpTargetPath) + 1] != 0)
+		if (lpTargetPath[_tcsnlen(lpTargetPath, ARRAYSIZE(lpTargetPath)) + 1] != 0)
```

Comment:
This way the `_tcsnlen` call is useless as it still might cause an out-of-bounds read if the buffer is not correctly terminated.

Reply:
good catch.